lib/list_module.rb

```diff
@@ -0,0 +1,7 @@
+module List
```

Comment:
`List` is too generic a name for this module.  It's often difficult to find a satisfactory name for this kind of thing but the one method it provides outputs a formatted list of dishes.